pyrene/src/components/Filter/FilterComponents/FilterBar.tsx

```diff
@@ -1,9 +1,44 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable react/static-property-placement */
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import styles from './FilterBar.css';
 import FilterPopoverButton from '../FilterPopOverButton/FilterPopoverButton';
 import FilterTag from './FilterTag';
+import {
+  FilterValues,
+  FilterValue,
+  Filter,
+  MultiSelectOption as Option,
+} from '../types';
+
+export interface FilterBarProps {
+  /**
+   * Sets the available filters.
+   * Type: [{ label: string (required), type: oneOf('singleSelect', 'multiSelect', 'text') (required), key: string (required), options: array of values from which user can choose in single/multiSelect}]
+   */
+  filters: Array<Filter>,
+  /**
+   * Filter values object.
+   * */
+  filterValues: FilterValues,
+  /**
+   * True to enable the visual components to handle negated filters.
+   */
+  negatable: boolean,
+  /**
+   * Called when the user clicks on the apply button. Exposes two parameters: filterValues and negatedFilterKeys (contains an array of the keys of the filters that are negated).
+   */
+  onFilterSubmit?: (filterValues: FilterBarProps['filterValues'], negatedKeys: Array<Filter['id']>) => void,
```

Comment:
what is the benefit of FilterBarProps['filterValues'] over just using FilterValues?